packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -214,6 +214,20 @@ class SharingCollection extends DocumentCollection {
       uri`/sharings/${sharing._id}/recipients/${recipientIndex}`
     )
   }
+
+  /**
+   * Revoke only one group of the sharing.
+   *
+   * @param {object} sharing Sharing Object
+   * @param {number} groupIndex Index of this group in the groups array of the sharing
+   */
+  revokeGroup(sharing, groupIndex) {
+    return this.stackClient.fetchJSON(
+      'DELETE',
+      uri`/sharings/${sharing._id}/groups/${groupIndex}`
+    )
+  }
```

Comment:
Can't we expose this method via the DSL? (aka destroy)

Reply:
I'm wondering if this makes sense because we're not deleting the io.cozy.sharing document, just modifying it. The other cases of revocation would have to be managed at the same time. That's how I'd imagine the api would look: 

**Revoke a group :** 
```js
client.collection('io.cozy.sharing').destroy(sharingDocument, {
  group: 1
})
```
**Revoke a recipient :** 
```js
client.collection('io.cozy.sharing').destroy(sharingDocument, {
  recipient: 0
})
```
Defining for revocation of all recipients bother me a little in terms of naming. Was it something like this that you saw ?

Reply:
I don't have any clue, I just know that since you are bypassing the store, you'll have to update the state manually. 

I'll try to move on being able to call any method from the query (client.query(Q('io.cozy.sharings').call('revokeGroup', {})), it should fix the issue.